egs2/voxceleb/spk1/conf/tuning/train_RawNet3_sgdr_bs512_da_sampler_adam.yaml

```diff
@@ -35,28 +40,34 @@ loss_conf:
   margin: 0.3
   scale: 30
 
-max_epoch: 40
+max_epoch: 160
 #num_iters_per_epoch: 1
 num_att_plot: 0
 num_workers: 8
 cudnn_deterministic: False
 cudnn_benchmark: True
-batch_size: 128
-#batch_type: unsorted
-iterator_type: sequence
-shuffle_within_batch: True
-log_interval: 50
-optim: adamw
+iterator_type: category
+valid_iterator_type: sequence
+shuffle_within_batch: False
+log_interval: 100
+batch_size: 512
```

Comment:
How about mentioning the GPUs and memory needed for using this configuration?

Reply:
Great idea! 

I reflected this to [a30783c](https://github.com/espnet/espnet/pull/5385/commits/a30783c7f38bd57329eb1c545dfd0a592422c32a) to be merged in the next PR, so that all CI should not run again for adding comments.